src/core/client/admin/test/components/UserHistoryDrawer/UserHistoryDrawer.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  createSettings,
+  createStory,
+  createUser,
+} from "coral-admin/../test/helpers/fixture";
+
+import { pureMerge } from "coral-common/utils";
+import { GQLResolver, GQLUser, GQLUSER_ROLE } from "coral-framework/schema";
+import {
+  act,
+  createResolversStub,
+  CreateTestRendererParams,
+  waitForElement,
+  within,
+} from "coral-framework/testHelpers";
+
+import create from "./create";
+
+const viewer = createUser();
+viewer.role = GQLUSER_ROLE.ADMIN;
+const settings = createSettings();
+
+async function createTestRenderer(
+  user: GQLUser,
+  params: CreateTestRendererParams<GQLResolver> = {}
+) {
+  const { testRenderer, context, subscriptionHandler } = create(
+    {
+      ...params,
+      resolvers: pureMerge(
+        createResolversStub<GQLResolver>({
+          Query: {
+            settings: () => settings,
+            viewer: () => viewer,
+            user: () => user,
+          },
+        }),
+        params.resolvers
+      ),
+      initLocalState: (localRecord, source, environment) => {
+        localRecord.setValue(true, "loggedIn");
+        if (params.initLocalState) {
+          params.initLocalState(localRecord, source, environment);
+        }
+      },
+    },
+    user
+  );
+
+  return { testRenderer, context, subscriptionHandler };
+}
+
+it("User drawer is open for user, user name is visible", async () => {
+  const story = createStory();
+  const user = story.comments.edges[0].node.author!;
+  await act(async () => {
+    const { testRenderer } = await createTestRenderer(user);
+    const { getByText } = within(testRenderer.root);
+    await waitForElement(() => getByText(user.id, { exact: false }));
+  });
+});
+
+it("User drawer is open for user, user name is visible", async () => {
+  const story = createStory();
+  const user = story.comments.edges[0].node.author!;
+  await act(async () => {
+    const { testRenderer } = await createTestRenderer(user);
```

Comment:
test render should not be inside an `act` method. Refer to other tests as to where to use it.

Reply:
Makes sense, fixed these up.